src/editor/mouse.rs

```diff
@@ -41,6 +42,19 @@ impl Editor {
                     MouseLocation::Out => (),
                 }
             },
+            MouseEventKind::ScrollDown | MouseEventKind::ScrollUp => {
+                match self.find_mouse_location(event) {
+                    MouseLocation::File(_) => {
+                        let y = &mut self.doc_mut().offset.y;
```

Comment:
Note: this will cause issues by allowing the cursor to move out of bounds

Might be better to use the `move_down` method in the `Editor` struct